app/uk/gov/hmrc/thirdpartyapplication/scheduled/BCryptPerformanceMeasureJob.scala

```diff
@@ -45,9 +47,9 @@ class BCryptPerformanceMeasureJob @Inject() (logger: LoggerLike) extends Exclusi
     logger.info("[bcrypt Performance] Starting performance measurement")
 
     val timings = workFactorRangeToTest.map(workFactor => {
-      val startTime = DateTimeUtils.now.getMillis
+      val startTime = Instant.now().truncatedTo(MILLIS).toEpochMilli()
```

Comment:
I seem to recall a timeFuture function (or similar) in the core domain lib.

Reply:
Used timeThis() from SimpleTimer